apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -8,9 +10,21 @@ type Properties = {
 };
 
 const Sidebar = ({ items }: Properties): JSX.Element => {
+	const { permissions } = useAppSelector(({ auth }) => auth);
+
+	const filteredItems = items.filter((item) => {
+		if (item.label === "Access Management") {
+			return permissions?.some(
+				(permission) => permission.key === Permission.MANAGE_USER_ACCESS,
+			);
+		}
```

Comment:
It'll be better to create some `map`, which will map page name to an array of requeired permissions. Also use (create if it's not exist) enum for pages names instead of hardcoded string literal (do not forget to use this enum in `navigation-items.constant.ts`)